index.js

```diff
@@ -12,7 +12,11 @@ try {
   var EventEmitter = require('events').EventEmitter;
   if (!EventEmitter) throw new Error();
 } catch (err) {
-  var Emitter = require('emitter');
+  try {
+    var Emitter = require('emitter');
+  } catch (err) {
+    console.error('Unable to create an emitter:', err);
```

Comment:
This should not proceed if neither package was able to be loaded.

Reply:
I am now `throw`ing if no emitter can be created. If you'd prefer a different way to handle this, happy to work on it with you.

Reply:
It's perfectly fine 👍